nemoguardrails/library/domain_hallucination/extractors.py

```
@@ -0,0 +1,382 @@
+    "codespaces",
+    "site",
+    "contact",
+}
+
+_TLD_EXTRACTOR = None
+try:
+    import tldextract
+
+    _TLD_EXTRACTOR = tldextract.TLDExtract(suffix_list_urls=None, cache_dir=None)
+except Exception:
+    pass
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)